payment.yaml

```diff
@@ -0,0 +1,187 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Payment Ecosystem
+  description: Internal API that supplies abstraction over blockchain
```

Comment:
supplies abstraction?

Reply:
provides payment and wallet creation services